stripe/src/main/java/com/stripe/android/view/CardInputView.java

```diff
@@ -296,18 +308,26 @@ private void updateCvc(@NonNull @Card.CardBrand String brand) {
 
     private void updateScrollToPosition() {
         if (!mCardNumberIsViewed) {
-            int endScrollIndex = mIsAmex ? END_INDEX_AMEX : END_INDEX_COMMON;
+            int endScrollIndex = mIsAmEx ? END_INDEX_AMEX : END_INDEX_COMMON;
             String cardString = mCardNumberEditText.getText().toString();
             if (cardString.length() >= endScrollIndex) {
                 String hiddenString = cardString.substring(0, endScrollIndex);
-                mScrollToPostion =
-                        (int) Layout.getDesiredWidth(hiddenString, mCardNumberEditText.getPaint());
+                mScrollToPosition = mCustomWidthSetter == null
```

Comment:
The reason we need to use this hacky `mCustomWidthSetter` object is because in the test harness, the width of the string (and indeed the width of all of the controls) is 0, so nothing ever gets scrolled.